### ISSUE TITLE
window calls read with third argument

### ISSUE THREAD

**Issue by regular** [dominictarr/pull-window#1]
Hi Dominic,

in my (still ongoing) effort to fully grasp what `window` does, I stumbled across [these lines](https://github.com/dominictarr/pull-window/blob/1e42fed694e34933297914a3a4489997f2c24d65/index.js#L53).

Here you call `read` with a third argument. Isn't the signature `read(abort, cb)`, _always_? To see what it does, I removed the third argument and all tests still pass. Is this a left-over or am I missing something?

Thanks!

**Comment by dominictarr** [comment]
`next` comes from looper, https://github.com/dominictarr/pull-window/blob/1e42fed694e34933297914a3a4489997f2c24d65/index.js#L21

hmm, that doesn't make any sense at all. obviously it's calling next and not the callback though.

**Comment by regular** [comment]
yes, that additional function is just ignored ... Or maybe there is some special kind of source that uses a second callback? It's weird. Anyway, everything still seems to work fine if that code simply is deleted.
